src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -64,6 +64,7 @@ public static int Start(string consoleFilePath, [MarshalAs(UnmanagedType.LPArray
                 System.Diagnostics.Debugger.Break();
```

Comment:
@TravisEz13, your last commit had 81 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

TestGetContentWriter
```powershell
System.TypeInitializationException : The type initializer for 'System.Management.Automation.Runspaces.InitialSessionState' threw an exception.
---- System.NullReferenceException : Object reference not set to an instance of an object.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1587
   at PSTests.Parallel.FileSystemProviderTests.GetExecutionContext() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 51
   at PSTests.Parallel.FileSystemProviderTests.GetProvider() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 59
   at PSTests.Parallel.FileSystemProviderTests.TestGetContentWriter() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 175
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState..cctor() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 4550
```
TestClearProperty
```powershell
System.TypeInitializationException : The type initializer for 'System.Management.Automation.Runspaces.InitialSessionState' threw an exception.
---- System.NullReferenceException : Object reference not set to an instance of an object.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1587
   at PSTests.Parallel.FileSystemProviderTests.GetExecutionContext() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 51
   at PSTests.Parallel.FileSystemProviderTests.GetProvider() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 59
   at PSTests.Parallel.FileSystemProviderTests.TestClearProperty() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 152
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState..cctor() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 4550
```
TestGetContentReader
```powershell
System.TypeInitializationException : The type initializer for 'System.Management.Automation.Runspaces.InitialSessionState' threw an exception.
---- System.NullReferenceException : Object reference not set to an instance of an object.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1587
   at PSTests.Parallel.FileSystemProviderTests.GetExecutionContext() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 51
   at PSTests.Parallel.FileSystemProviderTests.GetProvider() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 59
   at PSTests.Parallel.FileSystemProviderTests.TestGetContentReader() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 162
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState..cctor() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 4550
```
TestClearContent
```powershell
System.TypeInitializationException : The type initializer for 'System.Management.Automation.Runspaces.InitialSessionState' threw an exception.
---- System.NullReferenceException : Object reference not set to an instance of an object.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1587
   at PSTests.Parallel.FileSystemProviderTests.GetExecutionContext() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 51
   at PSTests.Parallel.FileSystemProviderTests.GetProvider() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 59
   at PSTests.Parallel.FileSystemProviderTests.TestClearContent() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 189
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState..cctor() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 4550
```
TestSetProperty
```powershell
System.TypeInitializationException : The type initializer for 'System.Management.Automation.Runspaces.InitialSessionState' threw an exception.
---- System.NullReferenceException : Object reference not set to an instance of an object.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1587
   at PSTests.Parallel.FileSystemProviderTests.GetExecutionContext() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 51
   at PSTests.Parallel.FileSystemProviderTests.GetProvider() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 59
   at PSTests.Parallel.FileSystemProviderTests.TestSetProperty() in D:\a\1\s\test\xUnit\csharp\test_FileSystemProvider.cs:line 137
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState..cctor() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 4550
```